app/src/pages/Series.tsx

```diff
@@ -43,51 +45,55 @@ export const Series = () => {
   }, [seriesid]);
 
   if (hasError) {
-    return (
-      <Container maxWidth="xl" style={{ textAlign: "center" }}>
-        <Box marginTop="4rem">
-          <Typography
-            component="h1"
-            variant="h5"
-            fontWeight="bold"
-            gutterBottom
-          >
-            {t("competition.error")}
-          </Typography>
-        </Box>
-      </Container>
-    );
+    return <PageNotFound />;
   }
 
   if (isLoading) {
-    return (
-      <Box sx={{ width: "100%" }}>
-        <LinearProgress />
-      </Box>
-    );
+    return <LoadingPageLinear />;
   }
 
-  //Currently the dates of the first competition in the list are used for displaying registration open and close times
-  const registrationOpen = new Date(competitionData[0].registration_open);
-  const registrationClose = new Date(competitionData[0].registration_close);
+  const registrationOpenDates = competitionData.map(
+    (data: { registration_open: string }) => new Date(data.registration_open),
+  );
+  const isRegistrationDifferent = !registrationOpenDates.every(
+    (item: any) => item.getTime() === registrationOpenDates[0].getTime(),
+  );
+  const registrationOpen = new Date(Math.min(...registrationOpenDates));
+  const registrationClose = new Date(
+    Math.min(
+      ...competitionData.map(
+        (data: { registration_close: string }) =>
+          new Date(data.registration_close),
+      ),
+    ),
+  );
```

Comment:
```suggestion
  const earliestRegistrationClose = new Date(
    Math.min(
      ...competitionData.map(
        (data: { registration_close: string }) =>
          new Date(data.registration_close),
      ),
    ),
  );
```

And for consistency with the open dates, a `const registrationCloseDates` might be nice.